thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/EntityManagerResource.java

```diff
@@ -182,6 +185,9 @@ public Response updateEntity(@QueryParam("entityType") String entityTypeStr, Str
         break;
       case ALERT_CONFIG:
         AlertConfigDTO alertConfigDTO = OBJECT_MAPPER.readValue(jsonPayload, AlertConfigDTO.class);
+        if (Strings.isNullOrEmpty(alertConfigDTO.getFromAddress())) {
+          alertConfigDTO.setFromAddress(config.getFailureToAddress());
```

Comment:
Do we want to use "getFromAddress()" to setFromAddress()? It can be a little bit confusing here we are using a "toAddr" to set "fromAddr". Same thing for EmailResource line 65.

Reply:
thats toAddr from thirdeyeConfig which is te-dev address. Not same as subsGroup's toAddr.